### PR TITLE
Improved code of pipeline page by placing search bar in header(#2cxrd4t)

### PR DESCRIPTION
**Pull request by azkyakhan**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)
**Before:**
search-desktop-before

search-mobile-before

**After:**
search-desktop-after

search-mobile-after

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [X] I read and followed [contribution rules](https://github.com/hotwax/job-manager#contribution-guideline)

### CODE REVIEW

**Comment by azkyakhan** [comment]
https://user-images.githubusercontent.com/82817616/169263223-c55c3818-2a1c-44c8-b7db-b8756d99cabb.mp4

File: src/views/Pipeline.vue [hunk 1]
```diff
@@ -471,7 +471,7 @@ export default defineComponent({
               text: this.$t("CANCEL"),
               handler: async () => {
                 await this.store.dispatch('job/cancelJob', job);
-                await this.getPendingJobs();
+                await this.store.dispatch('job/fetchPendingJobs', {eComStoreId: this.getCurrentEComStore.productStoreId, viewIndex: 0, queryString: this.queryString});
```
Comment (adityasharma7) [inline src/views/Pipeline.vue:474]: Please revert this change

Comment (azkyakhan) [reply src/views/Pipeline.vue]: I have reverted this change

**Review by adityasharma7 — APPROVED** [review]

**Review by adityasharma7 — APPROVED** [review]
